### ISSUE TITLE
Possible to fall through nearby region tiles

### ISSUE THREAD

**Issue by alexchandel** [JapaMala/armok-vision#54]
In FPS mode, walking to the border of the map and attempting to jump onto a neighboring region causes one to fall off the world. The neighboring region tiles' floors, though contiguous with the map, appear to have no collision properties.

**Comment by JapaMala** [comment]
Yeah, I never really got around to adding collisions to them. I'll try to get it done in the next release.
